samples/experimental/tracing-client.js

```diff
@@ -0,0 +1,46 @@
+import http from "k6/http";
```

Comment:
I think we should move this file in `samples/experimental/tracing/` folder

Reply:
Absolutely, completely forgot to do it indeed 🤦🏻